src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -160,9 +179,15 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
         material.pushLayer(paramMaterial);
         const imageryLayers = context.view.getLayers(l => l.type === 'color');
         const sequence = ImageryLayers.getColorLayersIdOrderedBySequence(imageryLayers);
-        material.setSequence(sequence);
+        // do not update fragment shader yet
+        material.setSequence(sequence, false);
 
         initNodeImageryTexturesFromParent(node, node.parent, layer);
+
+        // do it now - after we set the atlas textures
+        material._updateFragmentShader();
```

Comment:
IMHO it's private to itowns, no to `LayeredMaterial`, hence the `_` prefix.

But I can remove it if you prefer.